daemon/daemon_main.go

```diff
@@ -599,6 +599,24 @@ func init() {
 	flags.Int(option.CTMapEntriesGlobalAnyName, option.CTMapEntriesGlobalAnyDefault, "Maximum number of entries in non-TCP CT table")
 	option.BindEnvWithLegacyEnvFallback(option.CTMapEntriesGlobalAnyName, "CILIUM_GLOBAL_CT_MAX_ANY")
 
+	flags.Duration(option.CTMapEntriesTimeoutTCPName, 21600*time.Second, "Timeout for established entries in TCP CT table")
```

Comment:
@borkmann for readability this can be `6*time.Hour` and `time.Minute` for the `60*time.Second`

Reply:
True, I left it as seconds here to have a better comparison wrt the defines from C which got removed.